src/js/components/Calendar/utils.js

```diff
@@ -139,15 +140,20 @@ export const getTimestamp = (date) =>
 // can create undesired results. The standardizes the input value
 // for internal calculations
 // Reference: https://www.ursahealth.com/new-insights/dates-and-timezones-in-javascript
-export const normalizeForTimezone = (value, timestamp) => {
+
+// default this to true
+export const normalizeForTimezone = (value, timestamp, normalize = true) => {
```

Comment:
So if normalize is false this degenerates to just convert the value toISOString()

Reply:
yes, I can add a comment so that's documented.

> I think it looks good. I'm trying to think of how to simplify this in general but it's hard to. I'd rather see the normalize flag not go so deep through so many calls and isolate it more to the initial look at defaultValue and value.

Agreed it's not the cleanest to have to pass `normalize` in so many calls, but seemed like the most reliable solution for the time being. Eric and I had explored passing this "normalize" concept in the `timestamp` variable potentially (as opposed to adding another parameter to certain functions) but this caused issues because of the way timestamp is used to append to the date in various places.

> Really it feels like the crux of the problem is caring about timezone at all when all we're trying to do is choose a day/month/year. That seems to boil down to how we accept an iso8601 string with all that info in it and try to maintain that format through the api even though we don't really use the time aspect or intentionally convert between timezones. Changing that would probably be breaking.

Yes, difficult to not introduce breaking changes.